README.md

```diff
@@ -52,117 +36,48 @@ supported.
 
 There is one exception - the documentation for the rules and cmdlets will remain in the [docs](docs)
 folder to facilitate build testing and to be archived as part of each release. Only the
 documentation for the latest release is published on on docs.microsoft.com.
 
-## Usage
-
-```powershell
-Get-ScriptAnalyzerRule [-CustomRulePath <String[]>] [-RecurseCustomRulePath] [-Name <String[]>] [-Severity <String[]>] [<CommonParameters>]
-
-Invoke-ScriptAnalyzer [-Path] <String> [-CustomRulePath <String[]>] [-RecurseCustomRulePath]
- [-IncludeDefaultRules] [-ExcludeRule <String[]>] [-IncludeRule <String[]>] [-Severity <String[]>] [-Recurse]
- [-SuppressedOnly] [-Fix] [-EnableExit] [-Settings <Object>] [-SaveDscDependency] [-ReportSummary] [-WhatIf]
- [-Confirm] [<CommonParameters>]
-
-Invoke-ScriptAnalyzer [-Path] <String> [-CustomRulePath <String[]>] [-RecurseCustomRulePath]
- [-IncludeDefaultRules] [-ExcludeRule <String[]>] [-IncludeRule <String[]>] [-Severity <String[]>] [-Recurse]
- [-IncludeSuppressed] [-Fix] [-EnableExit] [-Settings <Object>] [-SaveDscDependency] [-ReportSummary] [-WhatIf]
- [-Confirm] [<CommonParameters>]
-
-Invoke-ScriptAnalyzer [-ScriptDefinition] <String> [-CustomRulePath <String[]>] [-RecurseCustomRulePath]
- [-IncludeDefaultRules] [-ExcludeRule <String[]>] [-IncludeRule <String[]>] [-Severity <String[]>]
- [-IncludeSuppressed] [-EnableExit] [-Settings <Object>] [-SaveDscDependency] [-ReportSummary] [-WhatIf]
- [-Confirm] [<CommonParameters>]
-
-Invoke-ScriptAnalyzer [-ScriptDefinition] <String> [-CustomRulePath <String[]>] [-RecurseCustomRulePath]
- [-IncludeDefaultRules] [-ExcludeRule <String[]>] [-IncludeRule <String[]>] [-Severity <String[]>]
- [-SuppressedOnly] [-EnableExit] [-Settings <Object>] [-SaveDscDependency] [-ReportSummary] [-WhatIf]
- [-Confirm] [<CommonParameters>]
-
-Invoke-Formatter [-ScriptDefinition] <String> [[-Settings] <Object>] [[-Range] <Int32[]>] [<CommonParameters>]
-```
-
-[Back to ToC](#table-of-contents)
-
 ## Installation
 
-### From PowerShell Gallery
+To install **PSScriptAnalyzer** from the PowerShell Gallery, see
+[Installing PSScriptAnalyzer](https://docs.microsoft.com/powershell/utility-modules/psscriptanalyzer/overview#installing-psscriptanalyzer).
 
-```powershell
-Install-Module -Name PSScriptAnalyzer
-```
-
-**Note**: For PowerShell version `5.1.14393.206` or newer, before installing PSScriptAnalyzer,
-please install the latest NuGet provider by running the following in an elevated PowerShell session.
-
-```powershell
-Install-PackageProvider NuGet -MinimumVersion 2.8.5.201 -Force
-Exit
-```
+To install **PSScriptAnalyzer** from source code:
 
-#### Supported PowerShell Versions and Platforms
+### Requirements
 
-- Windows PowerShell 3.0 or greater
-- PowerShell Core 7.0.3 or greater on Windows/Linux/macOS
-- Docker (tested only using Docker Desktop on Windows 10 1809)
```

Comment:
I cant seem to find thedocker information in the doc pages now? Is it outdated and thus why it was removed?

Reply:
The supported platforms information is in the Overview article in docs. All of the discussion of the docker containers was for the supported versions of PowerShell. That information is in the PowerShell setup docs and doesn't really belong here. See https://docs.microsoft.com/powershell/scripting/install/powershell-in-docker